buildbot-buildmaster/README.md

```diff
@@ -0,0 +1,30 @@
+# buildbot-buildmaster
+This project contains dockerized services, serving a buildbot buildmaster to the public web over tls.
+It is intended to be used on a linux server.
+Currently, it runs under Debian Buster.
+## Setup
+- clone this repository to your server. 
+- install docker and docker-compose.
+- create a dedicated, unprivileged user for this project
+- add the user to the docker group
+- clone this repository to your server. You only need the folder containing this README.
+  - optional: create a branch for your secrets (git checkout -b secrets)
+- follow the setup instructions in the subfolders. Start with traefik and then buildbot.
```

Comment:
If there is a certain order that needs to be obeyed, please add an enumeration or number the subfolders/files accordingly.

Reply:
Added enumeration.